packages/react-cosmos-utils/src/unserializable-parts.js

```diff
@@ -28,20 +28,36 @@ const isSerializable = obj => {
 };
 
 const splitUnserializableParts = obj => {
-  // TODO: Perform nested separation of unserializable parts. Required if props
-  // will move to fixture.props https://github.com/react-cosmos/react-cosmos/issues/217
-  const serializable = {};
-  const unserializable = {};
+  const splitResult = {
+    serializable: { },
+    unserializable: { }
+  };
+
+  const { serializable, unserializable } = splitResult;
 
   Object.keys(obj).forEach(key => {
     if (isSerializable(obj[key])) {
       serializable[key] = obj[key];
+    } else if (key === 'props' && isPlainObject(obj[key])) {
+      Object.keys(obj.props).forEach(propKey => {
+        let propHome = 'serializable';
```

Comment:
But it would be wise to add the new tests first and then beautify the implementation.